src/VisualStudio/Core/Def/Implementation/ColorSchemes/ColorSchemeApplier.ClassificationVerifier.cs

```diff
@@ -0,0 +1,212 @@
+            private const uint DarkThemePlainText = 0x00DCDCDCu;
+            private const uint DarkThemeIdentifier = DarkThemePlainText;
+            private const uint DarkThemeOperator = 0x00B4B4B4u;
+            private const uint DarkThemeKeyword = 0x00D69C56u;
+
+            private const uint LightThemePlainText = 0x00000000u;
+            private const uint LightThemeIdentifier = LightThemePlainText;
+            private const uint LightThemeOperator = LightThemePlainText;
+            private const uint LightThemeKeyword = 0x00FF0000u;
+
+            private const string PlainTextClassificationTypeName = "plain text";
+
+            // Dark Theme Core Classifications
+            private static ImmutableDictionary<string, uint> DarkThemeForeground =>
+                new Dictionary<string, uint>()
+                {
+                    [PlainTextClassificationTypeName] = DarkThemePlainText,
+                    [ClassificationTypeNames.Identifier] = DarkThemeIdentifier,
+                    [ClassificationTypeNames.Keyword] = DarkThemeKeyword,
+                    [ClassificationTypeNames.Operator] = DarkThemeOperator,
+                }.ToImmutableDictionary();
+
+            // Light, Blue, or AdditionalContrast Theme Core Classifications
+            private static ImmutableDictionary<string, uint> BlueLightThemeForeground =>
+                new Dictionary<string, uint>()
+                {
+                    [PlainTextClassificationTypeName] = LightThemePlainText,
+                    [ClassificationTypeNames.Identifier] = LightThemeIdentifier,
+                    [ClassificationTypeNames.Keyword] = LightThemeKeyword,
+                    [ClassificationTypeNames.Operator] = LightThemeOperator,
+                }.ToImmutableDictionary();
+
+            // The High Contrast theme is not included because we do not want to make changes when the user is in High Contrast mode.
+
+            private IVsFontAndColorStorage? _fontAndColorStorage;
+            private IVsFontAndColorUtilities? _fontAndColorUtilities;
+
+            private ImmutableArray<string> Classifications { get; }
+
+            public ClassificationVerifier(IThreadingContext threadingContext, IServiceProvider serviceProvider, ImmutableDictionary<SchemeName, ColorScheme> colorSchemes)
+                : base(threadingContext)
+            {
+                _serviceProvider = serviceProvider;
+
+                _colorSchemes = colorSchemes.ToImmutableDictionary(
+                    nameAndScheme => nameAndScheme.Key,
+                    nameAndScheme => nameAndScheme.Value.Themes.ToImmutableDictionary(
+                        theme => theme.Guid,
+                        theme => theme.Category.Colors
+                            .Where(color => color.Foreground.HasValue)
+                            .ToImmutableDictionary(color => color.Name, color => color.Foreground!.Value)));
```

Comment:
❓ why is the `!.` needed here? If it's a `Nullable<T>` you shouldn't need

Reply:
Compiler complained.